src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -22,11 +22,33 @@ const SoSpecialWrapper = styled.div`
         margin-bottom: 3rem;
     }
     
-    .slick-arrow::before{
-        color: black;
-        font-size: 22px;
+    .slick-next{
+        right: -38px;
+    }
+    .slick-list{
+        padding-top: 2px;
+        text-align: center;
+    }
+    @media screen and (max-width: 992px) {
+        .slick-list{
+            margin: 0;	
+        }
+    }
+
+    .slick-arrow{
+        width: 4.8rem;	
+        height: 5rem;
+    }	
+    .slick-arrow:before{	
+        color: ${props => props.theme.primaryColor};
+        font-size: 6rem;
+        display: inline-block;
+        height: 2rem;
```

Comment:
3rem

Reply:
All changes have been made in the recent commit , thanks .